app/containers/MessageBox/index.js

```diff
@@ -181,14 +190,23 @@ class MessageBox extends Component {
 	// eslint-disable-next-line react/sort-comp
 	debouncedOnChangeText = debounce((text) => {
 		this.setInput(text);
+		const slashCommand = text.match(/^\/([a-z0-9._-]+) (.+)/im);
+		if (slashCommand) {
+			const [, name, params] = slashCommand;
```

Comment:
according to`const [, name, params] = slashCommand;` the `slashCommand` should be an array with `length === 2`. Consider checking `if (slashCommand.length === 2)`.

Reply:
it will be of length 3,  and match will return null  if not matched :)